rskj-core/src/main/java/co/rsk/net/eth/RskWireProtocol.java

```diff
@@ -272,19 +268,6 @@ public void sendStatus() {
                 status.getBestBlockNumber(),
                 channel.getPeerNodeID());
         sendMessage(rskmessage);
-
-        ethState = EthState.STATUS_SENT;
-    }
-
-
-    @Override
-    public boolean hasStatusPassed() {
```

Comment:
This was always false, since  `ethState` was never set to sth greater than `EthState.STATUS_SENT`.